src/threads.js

```diff
@@ -1174,6 +1171,32 @@ Process.prototype.errorObsolete = function () {
     throw new Error('a custom block definition is missing');
 };
 
+Process.prototype.errorBubble = function (error, element) {
+    // Return a morph containing an image of the elment causing the error
+    // above the text of error.
+    var errorMorph = new AlignmentMorph('column', 5),
```

Comment:
I love this design, Michael!

Reply:
thanks! 

Also, how do you feel about making the error speech bubbles "clickable"? I've been working with a couple folks, and one thing is that they disappear a little quickly.

Reply:
I'm open to suggestions and experiments!